nemo_skills/inference/model/tool_call.py

```
@@ -75,7 +95,7 @@ async def _execute_tool_call(self, tool_call, request_id: str):
         try:
             tool_args = json.loads(tool_args)
         except json.decoder.JSONDecodeError as e:
-            LOG.exception(e)
+            LOG.exception(f"Failed to parse tool arguments {tool_args}: {e}")
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)